README.md

```diff
@@ -1,2 +1,35 @@
 # dic
 dic - dic isn't conky
+
+# Concepts
+
+## Subsistems:
+
+### 1. Receiving telemetry through:
+
+* Unix-socket
+* Pipe
+* Return code of the third-party program
+
+### 2. Preparation of telemetry:
+
+* filtering
+* normalization
+* sorting
+* approx (?)
+* etc.
+
+### 3. Output to the screen:
+
+* plot
+* diagram
+* plain text
```

Comment:
add gauge and histogram to the list please, those are pretty essential
also it would be nice to note that whole point of output is to be able combine multiple "widgets" using some layout (e.g. grid)